mava/configs/env/lbf.yaml

```diff
@@ -0,0 +1,11 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - lbf_scenario: 2s-8x8-2p-2f-coop # [2s-8x8-2p-2f-coop, 8x8-2p-2f-coop, 2s-10x10-3p-3f, 10x10-3p-3f, 15x15-4p-3f, 15x15-4p-5f]
+# Further enviroment config details in "2s-8x8-2p-2f-coop" file.
+
+env_name: LevelBasedForaging-v0
+task_name: 2s-8x8-2p-2f-coop
```

Comment:
Similarly here, I think it is better to set the task names in the child configs to avoid logging mismatches.